client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-large-preview.tsx

```diff
@@ -193,9 +194,10 @@ const PatternLargePreview = ( {
 			frameRef={ frameRef }
 			onDeviceChange={ ( device ) => {
 				recordTracksEvent( PATTERN_ASSEMBLER_EVENTS.PREVIEW_DEVICE_CLICK, { device } );
+				setDevice( device );
 				// Wait for the animation to end in 200ms
+				// The animation is triggered by the setDevice above
 				window.setTimeout( () => {
-					setDevice( device );
```

Comment:
Moving `setDevice( device );` out of the timeout because the animation is triggered by it.